build.gradle

```diff
@@ -60,12 +61,14 @@ dependencies {
 
     //Web Connection Support
     api 'com.neovisionaries:nv-websocket-client:2.5'
-    api 'com.squareup.okhttp3:okhttp:3.12.0'
+    api 'com.squareup.okhttp3:okhttp:3.13.0-SNAPSHOT'
 
     //Opus library support
     api ('club.minnced:opus-java:1.0.4@pom') {
         transitive = true
     }
+    
+    compileOnly 'net.java.dev.jna:jna:4.4.0'
```

Comment:
Why this?

Reply:
Because its a runtime dependency of opus-java-api but we also use it for the Decoder class in JDA

Reply:
Was it not in the dependency tree before? This diff makes it look like it was added, not moved.

Reply:
Because it was added, we need it at compile-time for our class. Gradle only recently started actually enforcing this rule and it seems a bit inconsistent. Sometimes compilation fails and sometimes it works. This only ensures that JNA is present at compile-time but doesn't add it as an explicit dependency of JDA. We only need it for Decoder when opus-java is present anyway.

Reply:
`Gradle only recently started actually enforcing this rule and it seems a bit inconsistent.`

Okay, this makes sense. Thanks for the explanation.